apps/orams/pages/HomePage.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { withRouter, Switch, Route } from 'react-router-dom'
-import Homepage from 'orams/components/HomePage/HomePage'
+import Homepage from '../components/HomePage/HomePage'
```

Comment:
Why change from absolute path?

Reply:
It was failing on my branch, guess because my webpack wasn't setup. We can overwrite this.